src/contrib/custom_op/README.md

```diff
@@ -0,0 +1,92 @@
+# Adding Custom Operator in NNFusion
+
+## When do you need to add custom operator?
+1. There is an operator that NNFusion doesn't support yet;
```

Comment:
Shall we have an example to show this case (i.e., define the custom operator following the automatic frontend converter's format)?

Reply:
BTW, shall we add the doc link to the docs/ folder?